### DISCUSSION TITLE
Collaborations open?

### DISCUSSION THREAD

**Discussion by deniz946** [a1ex4/ownfoil#51] [General]
Hey, I've been using this project for several weeks now to have my own shop locally, and really like the project. Also been testing the v2 and the new UI looks very promising.

Been looking the code and trying to understand how the backend side works and I wanted to know if I could contact you @a1ex4 and help you with anything I can with this project, I love supporting and helping Open Source projects, and what a better way to return what you gave us with some help.

I'm a frontend engineer and I could help you with anything related to the frontend and with a little bit of time and onboarding, can also try to help you with the backend, not sure if you have some other people helping you or if you do accept support from other people, but really would like to :)

Thanks in advance!

**Comment by a1ex4** [comment]
Thanks for your interest in the project. Currently I have a very clear idea of where I want it to be, so I'm not looking to open it to collaboration just yet. I at least want to release v2 with all the base features, and then I think it will be easier for people to contribute.
